e2e-tests/spring-rest-openapi-v2/src/test/java/org/evomaster/e2etests/spring/examples/endpointFocusAndPrefix/EndpointFocusAndPrefixTest.java

```diff
@@ -0,0 +1,201 @@
+package org.evomaster.e2etests.spring.examples.endpointFocusAndPrefix;
+
+import com.foo.rest.examples.spring.endpointFocusAndPrefix.EndpointFocusAndPrefixController;
+import org.evomaster.client.java.instrumentation.shared.ClassName;
+import org.evomaster.core.problem.rest.RestIndividual;
+import org.evomaster.core.search.Solution;
+import org.evomaster.e2etests.spring.examples.SpringTestBase;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+
+public class EndpointFocusAndPrefixTest extends SpringTestBase {
+
+    private final String swaggerURLSample = "https://petstore.swagger.io/v2/swagger.json";
```

Comment:
in general, this is not a good idea, ie, to rely to external services for E2E tests. Wednesday afternoon hopefully i should have sometime to show you how to write these kinds of Spring APIs

Reply:
Fixed, and totally removed the external API. Only relies on the local application,